apps/sync/js/crypto/fxsyncwebcrypto.js

```diff
@@ -272,7 +271,7 @@ undle for collection ${collectionName}`);
     } catch(e) {
       return Promise.reject('Record cannot be JSON-stringified');
     }
-    const cleartext = StringConversion.rawStringToByteArray(cleartextStr);
+    const cleartext = StringConversion.utf16StringToByteArray(cleartextStr);
```

Comment:
utf16StringToByteArray does convert the string from UTF-16 to UTF-8 byte array. I will modify the function name to utf16StringToUtf8ByteArray.